### ISSUE TITLE
Optimize Images

### ISSUE THREAD

**Issue by Rahulm2310** [anitab-org/anitab-org.github.io#198]
### Describe the bug
A picture speaks thousand words, but if the picture is of large size, it takes forever to load.
Large images slow down the web pages which creates a less than optimal user experience.
We can reduce your image’s file sizes to help improve your website’s performance.

### To Reproduce
1. Go to https://anitab-org.github.io/
2. Right Click -> Inspect Element -> Go to Network Tab
3. Check out the file sizes and loading time

### Expected behavior
We can reduce the image sizes without compromising the image quality.

### Screenshots
Here's an example 
![anitab_issue](https://user-images.githubusercontent.com/54268438/105041392-f35de080-5a88-11eb-9cf7-5670219b2b9b.JPG)

Also, the about image is of 1.1 MB which is very large and takes very long to load in a poor internet connection and needs to be optimized.
<img width="671" alt="image-about" src="https://user-images.githubusercontent.com/54268438/105041889-8f87e780-5a89-11eb-9713-1dabbef0adbf.png">

### Desktop (please complete the following information):
 - OS: Windows 10
 - Browser : Chrome
 - Version : 87.0.4280.141 (Official Build) (64-bit)

### Additional context
I tried optimizing a image from the website and the size reduced significantly with no reduction in quality.
Before Optimization : Size - 470.8 KB
![landing 1c27c683](https://user-images.githubusercontent.com/54268438/105042349-2359b380-5a8a-11eb-85ee-1a68d8bb4a50.png)

After Optimization :  Size - 66.2 KB
![landing 1c27c683 (1)](https://user-images.githubusercontent.com/54268438/105042381-2eacdf00-5a8a-11eb-9c5f-7722dc64c72a.png)

![after-compression](https://user-images.githubusercontent.com/54268438/105042458-48e6bd00-5a8a-11eb-9deb-385d1634c457.JPG)

I want to work on this issue and will create a pull request asap. 
Please assign it to me.

**Comment by annabauza** [comment]
Please remeve comments like `I want to work on this issue` from description.

**Comment by Rahulm2310** [comment]
@annabauza removed it.

**Comment by annabauza** [comment]
@Rahulm2310 also remove from description `@annabauza Please share your thoughts 😊` other then that issue is approved

**Comment by Rahulm2310** [comment]
@annabauza now can it be assigned to me ?

**Comment by rutvi18** [comment]
Can I take up the issue?

**Comment by Rahulm2310** [comment]
@rutvi18 Please have a look on other available issues. I am already looking into it. Thanks

**Comment by annabauza** [comment]
Technically you have not been assigned @Rahulm2310 , however indeed you asked first to be assigned to this issue. In other hand you have created the issue, so I'm not sure here. @isabelcosta @mayburgos can you help me out here?

**Comment by Rahulm2310** [comment]
@annabauza I have not started working on it. I am just looking for good libraries we could use for image compression . 🙂

**Comment by isabelcosta** [comment]
@annabauza I would say @Rahulm2310 created this, and if this is not considered as "first timers only" then @Rahulm2310 can work on this, for asking for it first.

**Comment by annabauza** [comment]
Thank you @isabelcosta . @Rahulm2310 yours :)

**Comment by Rahulm2310** [comment]
@annabauza @isabelcosta Should we use something like [Imgbot](https://imgbot.net/).
Actually, there are not much image compression libraries for frontend. Most of them are for server side. Ex: [imagemin](https://www.npmjs.com/package/imagemin). Or, if you guys have a suggestions, please share your views

**Comment by annabauza** [comment]
I think designers should compress I'll rise this with design team.

**Comment by Rahulm2310** [comment]
@annabauza yes definitely they can, but I think we need a long term solution which could compress all images we add to the website in future.

**Comment by annabauza** [comment]
I understand your thinking here, but quality needs to be approved by human designer eye. This is why it's designer responsibility.

**Comment by nandini45** [comment]
@Rahulm2310  Can u tell us your progress about this issue??

**Comment by Rahulm2310** [comment]
@nandini45 I thought the design team will handle this issue. If we need manual optimization, I can do it within 1 hr and create a PR. Shall I ?

**Comment by Ayushi271** [comment]
Hello
My sincere greetings
Is this issue still open?
Can you assign it to me?

**Comment by aditi5050** [comment]
Hello @tinniaru3005  can I work on with this issue can you please assign this to me

**Comment by tinniaru3005** [comment]
Unassigned due to long inactivity so that others can take up the issue.

**Comment by tinniaru3005** [comment]
Hey @aditi5050, here you go! :)

**Comment by aditi5050** [comment]
@tinniaru3005 thanks alot :)
